src/main/java/build/buildfarm/worker/shard/Worker.java

```diff
@@ -387,6 +388,25 @@ public static int KBtoBytes(int sizeKb) {
     return sizeKb * 1024;
   }
 
+  private static Duration GetGrpcTimeout(ShardWorkerConfig config) {
```

Comment:
method names should start with lowercase - getGrpcTimeout.

Reply:
fixed in followup diff